Looped/Interface/Popular/PopularViewController.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+
+protocol PopularViewControllerFlowDelegate: class, DetailsShowable {}
```

Comment:
Ja bym zrezygnowal z protokolu 

```DetailsShowable``` on nic nie wniesie wystarczy flow delegate

Reply:
Trzy inne protokoły (PopularViewControllerFlowDelegate, User... i Create...) dziedzicząc po nim używają tej funkcji, nie muszę jej trzy razy deklarować. Warto to zostawić czy w każdym z protokołów zrobić tę funkcję?